### PR TITLE
Finish Advanced login

### PR DESCRIPTION
**Pull request by Kunjky**
image

### CODE REVIEW

**Comment by Kunjky** [comment]
ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -1,19 +1,27 @@
 class SessionsController < ApplicationController
+  before_action :fetch_user, only: [:create]
```
Comment (tranducquoc) [inline app/controllers/sessions_controller.rb]: remove []

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
   def new; end
 
   def create
-    user = User.find_by(email: params[:session][:email].downcase)
-    if user && user.authenticate(params[:session][:password])
-      log_in user
-      redirect_to user
+    if @user && @user.authenticate(params[:session][:password])
+      log_in @user
+      params[:session][:remember_me] == "1" ? remember(@user) : forget(@user)
```
Comment (tranducquoc) [inline app/controllers/sessions_controller.rb]: setting cho 1

**Comment by tranducquoc** [comment]
commented

**Comment by Kunjky** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**